### ISSUE TITLE
should test use of implicit derivatives in point rasterization mode

### ISSUE THREAD

**Issue by dneto0** [gpuweb/cts#1182]
In [point rasterization](https://gpuweb.github.io/gpuweb/#point-rasterization), it's not so obvious that there will be helper invocations to assist computation of implicit derivatives.  

The CTS should exercise this case, particularly the functions mentioned in https://gpuweb.github.io/gpuweb/wgsl/#derivatives

Sending to dj2 as part of the big bag of work to track.

**Comment by dneto0** [comment]
Kudos to @Kangz for raising this.

**Comment by dj2** [comment]
@jrprice will this be covered by the derivative tests you've been adding?

**Comment by jrprice** [comment]
No, I didn't consider different primitives and only tested with a single triangle.
